syntax = "proto3";

package test.envoy.config.xds;

// Configuration for InvalidProtoKeyValueStore.
message InvalidProtoKeyValueStoreConfig {
}
